proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -63,3 +65,11 @@ message UpdateOrganizationVerificationStatusRequest {
 }
 
 message UpdateOrganizationVerificationStatusResponse {}
+
+message CreateMachineUserRequest {
+  string username = 1;
+}
+
+message CreateMachineUserResponse {
+  User user = 1;
```

Comment:
Does the User object need an enum or a type to differentiate between a "regular" vs "machine" user?

Reply:
yes, but I am planning to include this in a later PR